tests/test_service.py

```diff
@@ -0,0 +1,368 @@
+from PyCrypCli.client import Client
+from PyCrypCli.exceptions import (
+    AlreadyOwnThisServiceException,
+    CannotDeleteEnforcedServiceException,
+    CannotToggleDirectlyException,
+    CouldNotStartService,
+    DeviceNotFoundException,
+    DeviceNotOnlineException,
+    InvalidRequestException,
+    PermissionDeniedException,
+    ServiceCannotBeUsedException,
+    ServiceNotFoundException,
+    ServiceNotSupportedException,
+    WalletNotFoundException,
+)
+
+from database import execute
+from testcase import TestCase
+from tests.test_device import setup_device
+from tests.test_server import setup_account, super_password, super_uuid
+from util import get_client, uuid
+
+
+def create_service(device=uuid(), name="telnet", n=1, part_owner=None, owner=super_uuid, clear_service=True):
```

Comment:
Don't call other functions in function definitions. As this is a required argument, remove the default value instead.